src/__tests__/ajv.test.ts

```diff
@@ -39,17 +41,18 @@ describe("tests for the ajv custom formatters", () => {
     },
   ];
 
-  it.each(valid_inputs)("$label", (inputWithLabel) => {
+  it.each(valid_inputs)("$label", async (inputWithLabel) => {
     expect.hasAssertions();
     must(ajv);
     const { label, ...input } = inputWithLabel;
-    const result = ajv.validate(
-      "https://asl-path-validator.cloud/example.json#",
-      input
+    const validator = ajv.getSchema(
+      "https://asl-path-validator.cloud/example.json#"
     );
+    must(validator);
+    const result = await validator(input);
```

Comment:
slight change in how async validations are done

we can't use the top level `validate` function on AJV since that it synchronous. Instead, we get the validator function for the schema and invoke it as an asynchronous function.

The result is the input when no error or it throws. This will be another required change on how this is used in asl-validator